de.metas.business/src/main/java-legacy/org/compiere/model/MPayment.java

```diff
@@ -1890,8 +1890,14 @@ private void checkCreditLimit()
 		}
 
 		final I_C_BPartner partner = InterfaceWrapperHelper.create(getCtx(), getC_BPartner_ID(), I_C_BPartner.class, get_TrxName());
-		final BPartnerStats stats = Services.get(IBPartnerStatsDAO.class).getCreateBPartnerStats(partner);
+		final IBPartnerStatsDAO bpartnerStatsDAO = Services.get(IBPartnerStatsDAO.class);
+		final BPartnerStats stats = bpartnerStatsDAO.getCreateBPartnerStats(partner);
```

Comment:
use the ` IBPartnerStatsDAO.getCreateBPartnerStats(final int bpartnerId)` u just introduced